### PR TITLE
USHIFT-1975: add check-payload test rpm scan to fips scenario

### PR DESCRIPTION
**Pull request by eslutsky**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -102,7 +102,7 @@ debug:
 	@echo PATCH:"$(PATCH)"
 
 # These tags make sure we can statically link and avoid shared dependencies
```
Comment (ggiguash) [inline Makefile:104]: Please, fix the comment as noted in the previous PR

Comment (eslutsky) [reply Makefile]: removed the comment

File: test/suites/fips/validate-fips.robot [hunk 2]
```diff
@@ -46,6 +51,15 @@ Teardown
     Wait For MicroShift
     Logout MicroShift Host
 
+Check Payload Tool Must Pass
+    [Documentation]    Run check-paylod Tool
+    ${podman_args}=    Set Variable    --authfile /etc/crio/openshift-pull-secret --privileged -i -v /:/myroot
+    ${scan_command}=    Set Variable    scan node --root /myroot
+    ${stdout}    ${stderr}    ${rc}=    Execute Command
+    ...    podman run ${podman_args} ${CHECK_PAYLOAD_IMAGE} ${scan_command}
+    ...    sudo=True    return_rc=True    return_stdout=True    return_stderr=True
```
Comment (ggiguash) [inline test/suites/fips/validate-fips.robot:60]: How much output is produced by this command? If it's more than 32K chars, we need to redirect it to a file and copy it elsewhere for examination.

Comment (eslutsky) [reply test/suites/fips/validate-fips.robot:60]: its less , the whole [output](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_microshift/2703/pull-ci-openshift-microshift-main-microshift-metal-tests/1733836177967943680/artifacts/microshift-metal-tests/openshift-microshift-e2e-metal-tests/artifacts/scenario-info/el92-src@fips/log.html#s1-s1-t2-k1-k3) is shown at the log

Comment (eslutsky) [reply test/suites/fips/validate-fips.robot]: I've redirected stdout and stderr to  artifact file in the CI.

**Comment by ggiguash** [comment]
Approving the PR and putting on hold in case more people would like to review. Feel free to unhold if no more reviews are pending.

/lgtm
/hold

**Comment by ggiguash** [comment]
The test failed on the known problem of Network Smoke Test
/override ci/prow/microshift-metal-tests

**Pull request merged**